src/OpenTelemetry.Exporter.Geneva/TLDExporter/TldLogExporter.cs

```diff
@@ -411,22 +411,26 @@ internal void SerializeLogRecord(LogRecord logRecord)
         partCFieldsCountFromState = dataForScopes.PartCFieldsCountFromState;
 
         int partCFieldsCount = partCFieldsCountFromState + hasEnvProperties; // We at least have these many fields in Part C
-        var partCFieldsCountPatch = eb.AddStruct("PartC", (byte)partCFieldsCount);
 
-        for (int i = 0; i < partCFieldsCountFromState; i++)
+        if (partCFieldsCount > 0)
```

Comment:
Is there an easy way to unit test this?

Reply:
Not that I know of. TldExporter would only export when the EventProvider is enabled. For some reason, configuring an `EventListener` does not make it happen.

I'll try to find more on this.